addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.js

```diff
@@ -58,6 +60,15 @@ export class ThreadViewTopbar extends Component {
         document.removeEventListener('click', this._onClickCaptureGlobal, true);
     }
 
+    _onUpdate() {
```

Comment:
missing docstring, but you could just as well just not create this method and inline this code inside `useUpdate` instead